yt_dlp/extractor/moviepilot.py

```diff
@@ -0,0 +1,121 @@
+        'url': 'https://www.moviepilot.de/movies/interstellar-2/',
+        'info_dict': {
+            # 'id': 'interstellar-2',
+            'id': 'x7xdut5',
+            'ext': 'mp4',
+            'title': 'Interstellar',
+            'thumbnail': 'https://s1.dmcdn.net/v/SaXev1VvzitVZMFsR/x720',
+            'timestamp': 1400491705,
+            'description': 'md5:7dfc5c1758e7322a7346934f1f0c489c',
+            # coming from Dailymotion
+            'uploader': 'Moviepilot',
+            'like_count': int,
+            'view_count': int,
+            'uploader_id': 'x6nd9k',
+            'upload_date': '20140519',
+            'duration': 140,
+            'age_limit': 0,
+            'tags': ['Alle Trailer', 'Movie', 'Third Party'],
+        },
+    }, {
+        'url': 'https://www.moviepilot.de/movies/interstellar-2',
+        'only_matching': True,
+    }, {
+        'url': 'https://www.moviepilot.de/movies/interstellar-2/kinoprogramm/berlin',
+        'only_matching': True,
+    }, {
+        'url': 'https://www.moviepilot.de/movies/queen-slim/trailer',
+        'info_dict': {
+            'id': 'x7xj6o7',
+            # 'id': 'queen-slim',
+            'title': 'Queen & Slim',  # test escaping
+            'ext': 'mp4',
+            'thumbnail': 'https://s2.dmcdn.net/v/SbUM71WtomSjVmI_q/x720',
+            'timestamp': 1571838685,
+            'description': 'md5:73058bcd030aa12d991e4280d65fbebe',
+            # coming from Dailymotion
+            'uploader': 'Moviepilot',
+            'like_count': int,
+            'view_count': int,
+            'uploader_id': 'x6nd9k',
+            'upload_date': '20191023',
+            'duration': 138,
+            'age_limit': 0,
+            'tags': ['Movie', 'Verleih', 'Neue Trailer'],
+        },
+    }, {
+        'url': 'https://www.moviepilot.de/movies/der-geiger-von-florenz/trailer',
+        'info_dict': {
+            'id': 'der-geiger-von-florenz',
+            'title': 'Der Geiger von Florenz',
+            'ext': 'mp4',
+        },
+        'skip': 'No trailer for this movie.',
+    }, {
+        'url': 'https://www.moviepilot.de/movies/muellers-buero/',
+        'info_dict': {
+            'id': 'x7xcw1i',
+            # 'id': 'muellers-buero',
+            'title': 'Müllers Büro',  # test umlauts
+            'ext': 'mp4',
+            'description': 'md5:57501251c05cdc61ca314b7633e0312e',
+            'timestamp': 1287584475,
+            'age_limit': 0,
+            'duration': 82,
+            'upload_date': '20101020',
+            'thumbnail': 'https://s1.dmcdn.net/v/SaMes1WfAm1d6maq_/x720',
+            'uploader': 'Moviepilot',
+            'like_count': int,
+            'view_count': int,
+            'tags': ['Alle Trailer', 'Movie', 'Verleih'],
+            'uploader_id': 'x6nd9k',
+        },
+    }]
+
+    def _get_property(self, name, webpage, fatal=True):
+        return self._search_regex(f'meta itemprop="{name}" content="(.*?)"', webpage, name, fatal=fatal)
```

Comment:
this already exists as `self._html_search_meta`

Reply:
Unfortunately, `self._html_search_meta` is too broad, at least for the `description` field.

For example, in the Interstellar wegpage the following two entities are present:
```
<meta name="description" content="Interstellar Trailer &amp; Teaser, Interviews, Clips und mehr Videos auf Deutsch und im Original. Schaue dir alle 35 Videos jetzt an!"> 
```
```
<meta itemprop="description" content="16 von 35 (Englisch, 2014), Originaltitle: Interstellar, Hard SF, Science Fiction-Film, Abenteuerfilm, von Christopher Nolan, mit Matthew McConaughey, Anne Hathaway" />
```

My function matches the (correct) second one, while `self._html_search_meta` matches also the first one.

Reply:
then use this only for description (no need to make a function) and add a comment like `_html_search_meta is not used since we don't want name=description to match`